src/lib/server/loadHomeData.ts

```diff
@@ -110,15 +111,36 @@ export const loadHomeData = async ({
     res.json(),
   ) as Promise<GetCommitDataResponse>;
 
-  const [news, events, upcomingMeeting, previousMeeting, cafeOpen, commitData] =
-    await Promise.allSettled([
-      newsPromise,
-      eventsPromise,
-      upcomingMeetingPromise,
-      previousMeetingPromise,
-      cafeOpenPromise,
-      commitPromise,
-    ]);
+  // ACTIVE MANDATE?
+  const hasActiveMandatePromise =
+    prisma.mandate.findFirst({
+      where: {
+        startDate: { lte: now },
+        endDate: { gte: now },
+        memberId: locals.user?.memberId,
+      },
+      select: {
+        id: true,
+      },
+    }) !== null;
```

Comment:
@alfredgrip Is this really correct? It looks to me like we're checking if `promise !== null`, since we're not awaiting the promise here but rather with Promise.allSettled.

Reply:
Great catch! Fixed in #536